conan/cli/printers/graph.py

```diff
@@ -98,6 +98,11 @@ def _format_resolved(title, reqs_to_print):
     if deprecated:
         output.warning("There are deprecated packages in the graph", warn_tag="risk")
 
+    if graph.visibility_conflicts:
+        output.warning("Packages required both with visible=True and visible=False", warn_tag="risk")
+        for ref, consumers in graph.visibility_conflicts.items():
+            output.info(f"    {ref}: Required by {', '.join(str(c) for c in consumers)}")
```

Comment:
Should this still be a warning? Or at least skippable with the same warn_tag than above?

Reply:
Good point. I was aiming to have a cleaner output, avoiding the "WARN: risk" at each line of the report.
I'll try to do a multiline warning instead.

Reply:
Done